playbook/app/pb_kits/playbook/pb_time_stacked/_time_stacked.html.erb

```diff
@@ -1,8 +1,12 @@
 <%= content_tag(:div,
-    align: object.align,
     id: object.id,
     data: object.data,
     class: object.classname) do %>
-      <%= pb_rails("body", props: { text: object.format_time_string, color: "light" }) %>
-      <%= pb_rails("caption", props: { text: object.format_timezone, color: "light" }) %>
+
+    <%= pb_rails("body", props: { color: "light", classname: "pb_time_stacked time-spacing" }) do %>
+      <time>
```

Comment:
I think the time itself can be added as an attribute on the time element if you wanted to do that, but I think I remember just doing the wrap like below is cool too. `<time datetime="${object.pb_date_time.etc}">` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time.